src/core/Akka.Tests.Shared.Internals/AkkaSpec.cs

```diff
@@ -39,13 +40,13 @@ public abstract class AkkaSpec : Xunit2.TestKit    //AkkaSpec is not part of Tes
 
         private static int _systemNumber = 0;
 
-        public AkkaSpec(string config)
-            : this(ConfigurationFactory.ParseString(config).WithFallback(_akkaSpecConfig))
+        public AkkaSpec(string config, ITestOutputHelper output = null)
```

Comment:
Ok, that makes sense - and it looks like this is only for XUnit2 TestKit and AkkaSpec. I originally misread what you wrote and thought that this was for the TestKit base class itself, which would be a much larger change obviously.

I'm good with this change. cc @rogeralsing ?